### PR TITLE
Deprecate settings, which are now given by the respective widgets

### PR DESCRIPTION
**Pull request by felixgehrmann**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1599,6 +1602,7 @@
                 "pagination.itemsPerPage": {
                     "type": "inputText",
                     "required": false,
+                    "deprecated": true,
```
Comment (dbohla) [inline config.json]: @stentrop Kam dieser Wert nicht erst in Verbindung mit dem ItemsPerPage-Widget rein?

Comment (stentrop) [reply config.json]: ja kam er, sollte also nicht deprecated werden

Comment (felixgehrmann) [reply config.json]: Setting wurde dedeprecated.

Comment (felixgehrmann) [reply config.json]: Setting wurde dedeprecated.

**Review by Dominik2809 — APPROVED** [review]

**Pull request merged**